index.bs

```diff
@@ -207,19 +207,21 @@ partial interface DocumentOrShadowRoot {
 		On setting, {{adoptedStyleSheets}} performs the following steps:
 
 		1. Let |adopted| be the result of converting the given value to a FrozenArray&lt;CSSStyleSheet>
-		2. If any entry of |adopted| has its [=constructed flag=] not set, throw a "{{NotAllowedError}}" {{DOMException}}. 
+
+		2. If any entry of |adopted| has its [=constructed flag=] not set, or its [=constructor document=] is not equal to this {{DocumentOrShadowRoot}}'s [=node document=], throw a "{{NotAllowedError}}" {{DOMException}}. 
+
 		3. Set this {{DocumentOrShadowRoot}}'s [=adopted stylesheets=] to |adopted|.
 	</dd>
 </dl>
 
 Every {{DocumentOrShadowRoot}} has <dfn>adopted stylesheets</dfn>.
 
 The user agent must include all style sheets in the {{DocumentOrShadowRoot}}'s
-[=adopted stylesheets=] whose [=constructor document=] is the same as the {{DocumentOrShadowRoot}}'s [=node document=] inside its <a>document or shadow root CSS style sheets</a>.
+[=adopted stylesheets=] inside its <a>document or shadow root CSS style sheets</a>.
 
 These [=adopted stylesheets=] are ordered after all the other style sheets (i.e. those derived from {{DocumentOrShadowRoot/styleSheets}}).
 
-<p class="note">
-Note that because the [=adopted stylesheets=] are a property of the {{DocumentOrShadowRoot}}, they move along with the {{ShadowRoot}} if it gets [=adopted=] into a different {{Document}}, e.g. when adopting its [=shadow host=]. However, only [=adopted stylesheets=] that have the [=constructor document=] equal to the new {{Document}} will be applied, which means that a constructed <a interface>CSSStyleSheet</a> is only applicable in the document tree of its [=constructor document=].
-</p>
+This specification defines the following [=adopting steps=] for {{ShadowRoot}} nodes, given |node| and <var ignore> oldDocument</var>:
```

Comment:
Why is the var ignored? Was it giving you an error because the extra space was tripping its typo detector?

Reply:
It isn't used by the algorithm

Reply:
Then why is it mentioned here at all?

Reply:
Ah, the adoption steps algo passes that in. Nevermind, then.

Reply:
Adopting steps take those two arguments. There's no precedent in web specs to say whether they're like JS (you can declare your callback with fewer args and if it's called with more it'll be ignored) or C++/C#/Java (you need to match your callback type to its caller).